### PR TITLE
Add Intro page of TCD

### PR DESCRIPTION
**Pull request by mhh0318**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@sayakpaul 

Hi Sayak, I've updated an intro page for TCD. Please have a review.

### CODE REVIEW

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:25]: (nit): Could we maybe make these bullet points?

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:21]: The reader doesn't yet know what NFE is. Also, it's worth hyperlinking DPM-Solver++(2S).

Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:21]:
```suggestion
> ***Better than Teacher:*** TCD maintains superior generative quality at both low NFEs and high NFEs, even exceeding the performance of DPM-Solver++(2S) with Stable Diffusion XL (SDXL). It is worth noting that no additional discriminator or LPIPS supervision is included during training.
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:29]:
```suggestion
Trajectory consistency distillation can be directly placed on top of a pre-trained diffusion model as a [LoRA](https://huggingface.co/docs/diffusers/main/en/training/lora) module. Such a LoRA can be identified as a versatile acceleration module applicable to different fine-tuned models or LoRAs sharing the same base model without the need for additional training.
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:33]:
```suggestion
The corresponding checkpoints can be found at [TCD-SD15](https://huggingface.co/h1t/TCD-SD15-LoRA), [TCD-SD21-base](https://huggingface.co/h1t/TCD-SD21-base-LoRA), and [TCD-SDXL](https://huggingface.co/h1t/TCD-SDXL-LoRA), respectively.
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:45]:
```suggestion
TCD-LoRA can be considered an advanced method compared with [LCM-LoRA](https://huggingface.co/docs/diffusers/main/en/using-diffusers/inference_with_lcm_lora). The guide of TCD-LoRA workflow is:
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:97]:
```suggestion
<Tip>

Eta (referred to as `gamma` in the paper) is used to control the stochasticity in every step.
A value of 0.3 often yields good results, where eta = 0 means determinstic and eta = 1 is identity to Multi-step Consistency Sampler (as well as LCMScheduler).
We recommend using a higher eta when increasing the number of inference steps.

</Tip>
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:79]: Could we try to make use of non-human characters? Since it's the official documentation, I am slightly concerned about the reception of this. 

WDYT? @yiyixuxu @stevhliu

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:101]:
```suggestion
As mentioned above, the TCD-LoRA is versatile for community models and plugins. To test-drive this, load a community fine-tuned base model [animagine-xl-3.0](https://huggingface.co/cagliostrolab/animagine-xl-3.0).
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:130]:
```suggestion
Furthermore, TCD-LoRA also supports LoRAs corresponding to other styles. Below is an example with [Papercut](https://huggingface.co/TheLastBen/Papercut_SDXL). To learn more about how to combine LoRAs, refer to [this guide](https://huggingface.co/docs/diffusers/tutorials/using_peft_for_inference#combine-multiple-adapters).
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (sayakpaul) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:208]:
```suggestion
For this example, you'll keep using the SDXL model and the TCD-LoRA for SDXL with depth and canny ControlNets.
```

**Review by sayakpaul — COMMENTED** [review]
Very useful guide. Thanks for writing it in such details!

Let's wait for @stevhliu's reviews before applying suggestions.

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (mhh0318) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md:79]: Let's replace it with a cute cat.

Comment (sayakpaul) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md]: Yeah works for me. Could you generate the results for those as well?

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:99]: Better to have the tips upfront where users get the full picture before running the code

```suggestion
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:101]:
```suggestion
## Community models
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:103]:
```suggestion
TCD-LoRA also works with many community finetuned models and plugins. For example, load the [animagine-xl-3.0](https://huggingface.co/cagliostrolab/animagine-xl-3.0) checkpoint which is a community finetuned version of SDXL for generating anime images.
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:132]:
```suggestion
TCD-LoRA also supports other LoRAs trained on different styles. For example, let's load the [TheLastBen/Papercut_SDXL](https://huggingface.co/TheLastBen/Papercut_SDXL) LoRA and fuse it with the TCD-LoRA with the [`~loaders.UNet2DConditionLoadersMixin.set_adapters`] method. 

> [!TIP]
> Check out the [Merge LoRAs](merge_loras) guide to learn more about efficient merging methods.
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:208]:
```suggestion
## Adapters
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:210]:
```suggestion
TCD-LoRA is very versatile, and it can be combined with other adapter types like ControlNets, IP-Adapter, and AnimateDiff.
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:212]:
```suggestion
<hfoptions id="adapters">
<hfoption id="ControlNet">
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:344]:
```suggestion
</hfoption>
<hfoption id="IP-Adapter">
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:396]:
```suggestion
</hfoption>
<hfoption id="AnimateDiff">
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [inline docs/source/en/using-diffusers/inference_with_tcd_lora.md:435]:
```suggestion
![](https://github.com/jabir-zheng/TCD/raw/main/assets/animation_example.gif)

</hfoption>
</hfoptions>
```

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (mhh0318) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md]: To avoid confusion with `NFE`, I have continued with the description as  `the inference step`.

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (mhh0318) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md]: For [animagine-xl-3.0](https://huggingface.co/cagliostrolab/animagine-xl-3.0) and IP-Adapter, the human characters is performing better. Do we need to forcefully replace these?

Comment (sayakpaul) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md]: I will let @stevhliu comment further here.

Comment (stevhliu) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md]: Should be ok for this example 👍

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (stevhliu) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md:157]: Looks like this section hasn't been moved yet! Take a look at my suggestion above :)

**Review by stevhliu — COMMENTED** [review]
Just need to move the inpainting section and then we should be ready to merge :)

File: docs/source/en/using-diffusers/inference_with_tcd_lora.md
Comment (mhh0318) [reply docs/source/en/using-diffusers/inference_with_tcd_lora.md]: Just moved it as a subtag of general task. ;)

**Review by sayakpaul — APPROVED** [review]
Thanks for your great contributions! @stevhliu could you review once and merge?

**Review by stevhliu — APPROVED** [review]
LGTM, thanks again for your awesome contribution! 🤗

**Comment by mhh0318** [comment]
Thanks for your great efforts and assistance, Steven and Sayak!

**Pull request merged**
